tcms/testcases/migrations/0006_merge_text_field_into_testcase_model.py

```diff
@@ -20,17 +20,17 @@ def convert_test_case_text(test_case_text):
 
 
 def forward_copy_data(apps, schema_editor):
-    TestCase = apps.get_model('testcases', 'TestCase')
-    TestCaseText = apps.get_model('testcases', 'TestCaseText')
-    HistoricalTestCase = apps.get_model('testcases', 'HistoricalTestCase')
+    test_case = apps.get_model('testcases', 'TestCase')
+    test_case_text = apps.get_model('testcases', 'TestCaseText')
+    historical_test_case = apps.get_model('testcases', 'HistoricalTestCase')
 
-    for test_case in TestCase.objects.all():
-        latest_text = TestCaseText.objects.filter(case=test_case.pk).order_by('-pk').first()
+    for test_case in test_case.objects.all():
```

Comment:
loop variable the same as the variable holding the model - could lead to hard to debug issues. Don't use underscores for the model names.